_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -26,6 +26,8 @@ acknowledgments:
     - Jean-Yves Moyen
   funding:
     - WAI-Tools
+  assets:
+    - JFK's We Choose the Moon speech excerpt is courtesy of NASA.
```

Comment:
```suggestion
    - JFK's "We Choose the Moon" speech excerpt is courtesy of NASA.
```
Or in italics? Not sure which applies best: https://www.scribbr.com/mla/titles/
(needs to be done on all of them)

Reply:
I'm going with the quotation marks... according to your source, that's what seems most appropriate for an excerpt